src/migrations/Migration20230428070005_create_items_table.ts

```diff
@@ -0,0 +1,40 @@
+import { Migration } from '@mikro-orm/migrations';
+
+export class Migration20230428070005 extends Migration {
+
+  async up(): Promise<void> {
+    await this.addSql(`
+      CREATE TABLE IF NOT EXISTS items (
+      item_guid UUID DEFAULT uuid_generate_v4(), 
+      item_name VARCHAR(100) NOT NULL, 
+      unit_price REAL NOT NULL, 
```

Comment:
can we do unsigned real?

Reply:
postgres does not have unsigned integer or real types from what I have found until now.